cookbook/case_studies/ml_training/house_price_prediction/house_price_predictor.py

```diff
@@ -96,38 +120,43 @@ def gen_houses(num_houses) -> pd.DataFrame:
 
 
 # %%
-# Split the data into train, val, and test datasets.
+# Data Preprocessing
+# =====================
+#
+# We will create `feature` and `target` columns and split them into train and test subsets.
+# We will also split the input DataFrame into train, validation, and test datasets.
```

Comment:
Present tense, please. Also, the first line isn't clear. I'm assuming the second line should suffice, if not, please rephrase the first sentence.